### ISSUE TITLE
Orion: Tax page not findable in settings

### ISSUE THREAD

**Issue by sierrab123** [ingenesis/shopp#3500]
The page isn't redirecting properly

Firefox has detected that the server is redirecting the request for this address in a way that will never complete.

```
This problem can sometimes be caused by disabling or refusing to accept cookies.
```

**Comment by sierrab123** [comment]
This did work correctly as I am still seeing the taxes that I set-up weeks ago impact the testing purchases that I am making currently.

**Comment by jonathandavis** [comment]
Fixed in #3616.
